vispy/app/backends/_qt.py

```diff
@@ -453,39 +527,59 @@ def sizeHint(self):
     def mousePressEvent(self, ev):
         if self._vispy_canvas is None:
             return
+        if hasattr(ev, 'pos'):
+            posx, posy = ev.pos().x(), ev.pos().y()
+        else:
+            # Compatibility for PySide6 / PyQt6
+            posx, posy = ev.position().x(), ev.position().y()
```

Comment:
It seems you do this a lot. Maybe you could make a helper function (near the top of the module) like `_get_event_xy(ev):` and put this `if/else` there and then reuse it everywhere you current have this if/else.

Reply:
Done.